barretenberg/ts/tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "declarationMap": true,
     "importHelpers": true,
     "resolveJsonModule": true,
```

Comment:
msgpackr didnt type check, but doesnt affect much. this makes compilation faster, too